src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -0,0 +1,47 @@
+package org.vaccineimpact.orderlyweb.customConfigTests
+
+import org.assertj.core.api.Assertions
+import org.junit.Before
+import org.junit.Test
+import org.openqa.selenium.By
+import org.openqa.selenium.support.ui.ExpectedConditions
+import org.vaccineimpact.orderlyweb.db.JooqContext
+import org.vaccineimpact.orderlyweb.test_helpers.giveUserGroupGlobalPermission
+import org.vaccineimpact.orderlyweb.test_helpers.insertUserAndGroup
+
+class RunWorkflowTests : SeleniumTest()
+{
+    @Before
+    fun setUp()
+    {
+        JooqContext().use {
+            insertUserAndGroup(it, "test.user@example.com")
+            giveUserGroupGlobalPermission(it, "test.user@example.com", "reports.run")
+        }
+
+        startApp("auth.provider=montagu")
+
+        loginWithMontagu()
+
+        val url = RequestHelper.webBaseUrl + "/run-workflow/"
+        driver.get(url)
+    }
+
+    @Test
+    fun `can view run workflow tab`()
+    {
+        val tab = driver.findElement(By.id("run-workflow-tab"))
+        Assertions.assertThat(tab.findElement(By.tagName("h2")).text).isEqualTo("Create workflow")
+        Assertions.assertThat(tab.findElement(By.tagName("p")).text).isEqualTo("Run workflow is coming soon")
+    }
+
+    @Test
+    fun `can view run workflow progress tab`()
+    {
+        driver.findElement(By.id("workflow-progress-link")).click()
```

Comment:
I'm guessing this is a hangover from the old run report tabs implementation, but since the tabs themselves are a vue component, it would be better to test switching tabs in a front end test.

Reply:
I understand this may not be needed but since we have selenium tests then I think it is okay to leave it to test rendered contents on that page.

Reply:
A front end test would run quicker and keep down the build time, and be more consistent with our other testing strategy of vue components..

Reply:
I'd keep in the test before this, that the component is rendered at all, but testing the functionality of vue components should really be done in the vue test suites.